.travis.yml

```diff
@@ -22,13 +23,25 @@ cache:
   - debian/node_modules
 
 deploy:
-  provider: packagecloud
-  repository: "headset-electron"
-  username: "headsetapp"
-  token: "${PACKAGECLOUD_TOKEN}"
-  dist: "ubuntu/trusty"
-  local_dir: build/installers
-  skip_cleanup: true
-  on:
-    branch: master
-    tags: true
+  - provider: packagecloud
+    repository: "headset-electron"
+    username: "headsetapp"
+    token: "${PACKAGECLOUD_TOKEN}"
+    dist: "ubuntu/trusty"
+    local_dir: build/installers
+    package_glob: "*.deb"
+    skip_cleanup: true
+    on:
+      branch: master
+      tags: true
+  - provider: packagecloud
+    repository: "headset-electron"
+    username: "headsetapp"
+    token: "${PACKAGECLOUD_TOKEN}"
+    dist: "fedora/26"
+    local_dir: build/installers
+    package_glob: "*.rpm"
+    skip_cleanup: true
+    on:
+      branch: master
+      tags: true
```

Comment:
Once it's tested that the repo is working, I can get instructions on how to add it manually. I'm not as familiar with `rpm` repositories but I don't think it's that hard.